.babelrc

```diff
@@ -1,9 +1,4 @@
 {
-  "presets": [
-    "@babel/preset-env",
-    "@babel/preset-react"
-  ],
-  "plugins": [
-    "@babel/plugin-proposal-class-properties"
-  ]
+  "presets": ["@babel/preset-env", "@babel/preset-react"],
+  "plugins": ["@babel/plugin-proposal-class-properties"]
 }
```

Comment:
please revert to original formatting so that this file is removed from this PR

Reply:
it is reverted, but now it is formatted in a different way. Don't know why, because I still have the same prettier config.